erpnext/hr/report/monthly_attendance_sheet/monthly_attendance_sheet.py

```diff
@@ -66,11 +71,11 @@ def execute(filters=None):
 		emp_att_map = {}
 		for parameter in group_by_parameters:
 			data.append([ "<b>"+ parameter + "</b>"])
-			record, aaa = add_data(emp_map[parameter], att_map, filters, holiday_map, conditions, default_holiday_list, leave_list=leave_list)
+			record, aaa = add_data(emp_map[parameter], att_map, holiday_map, filters, default_holiday_list, leave_list=leave_list)
 			emp_att_map.update(aaa)
 			data += record
 	else:
-		record, emp_att_map = add_data(emp_map, att_map, filters, holiday_map, conditions, default_holiday_list, leave_list=leave_list)
+		record, emp_att_map = add_data(emp_map, att_map, holiday_map, filters, default_holiday_list, leave_list=leave_list)
```

Comment:
```suggestion
		record, attendance = add_data(emp_map, att_map, holiday_map, filters, default_holiday_list, leave_list=leave_list)
```
Better variable names for more context?